### DISCUSSION TITLE
Frontend Monetor challenge errors

### DISCUSSION THREAD

**Discussion by 2div** [community/community#22528] [New to GitHub]
<p>Dear community,</p>
<p>kindly advise me on below error , what is the issue with the line in my style sheet.  your advise appreciated.</p>
<p>below i attached error message and my style file message.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/e/bed208fe8d4ef383bb231d640dfecb1d623b1888.jpeg" data-download-href="/uploads/short-url/re4IuozgkslaSTOJwOhruun6xHq.jpeg?dl=1" title="Er1" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/96177680/181071641-d908cde9-77b6-487c-a200-027583e20c8a.jpeg" alt="Er1" data-base62-sha1="re4IuozgkslaSTOJwOhruun6xHq" width="690" height="179" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/e/bed208fe8d4ef383bb231d640dfecb1d623b1888_2_690x179.jpeg, https://aws1.discourse-cdn.com/github/original/3X/b/e/bed208fe8d4ef383bb231d640dfecb1d623b1888.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/b/e/bed208fe8d4ef383bb231d640dfecb1d623b1888.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/e/bed208fe8d4ef383bb231d640dfecb1d623b1888_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Er1</span><span class="informations">969×252 23.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<img src="https://user-images.githubusercontent.com/96177680/181071642-2aef6b77-d3f2-455b-9deb-6149d80f5ed6.jpeg" alt="Er2" data-base62-sha1="pZ2EnFy58nbKfRdICaE7fw5bnSE" width="608" height="77">

**Comment by jfullstackdev** [comment, marked as answer]
<p><img src="https://user-images.githubusercontent.com/47092464/181071643-873a6dbc-4bbc-4b9e-a2b2-73fc3de9610c.png" alt="image" data-base62-sha1="tNlQM19T56zpoDQAazbCmiXfSAf" width="288" height="27"></p>
<p>why is there a space at the filename? that will really be an error,</p>

**Comment by 2div** [comment]
<p>Thank for your advise. Its work now.</p>
